test-irregex-gauche.scm

```diff
@@ -1,1001 +0,0 @@
-
-;; derived from Gauche's regex tests
```

Comment:
So this is _not_ a version of the test suite ported to Gauche, but the Gauche native regex test suite ported to irregex, i.e. these tests are not duplicated in any other files so should be preserved.